x-pack/plugins/task_manager/server/saved_objects/schemas/task.ts

```diff
@@ -7,6 +7,26 @@
 
 import { schema } from '@kbn/config-schema';
 
+const SECONDS_REGEX = /^[1-9][0-9]*s$/;
```

Comment:
Can we use one of the functions from here, instead? x-pack/plugins/task_manager/server/lib/intervals.ts

Reply:
Updated in https://github.com/elastic/kibana/pull/204413/commits/7413a36de02d3153a1b5bc260f0a205d4891d4bd